### ISSUE TITLE
Pass phone numbers with +1 associated with them

### ISSUE THREAD

**Issue by nickoneill** [5calls/5calls#259]
Some people need the +1 to dial out of where ever they are and it hurts nobody (afaik), we should send these from the server rather than appending +1 like we do in web

**Comment by greggraf** [comment]
@nickoneill - have you done much work here? I had some ideas here and wanted to to take a crack.

**Comment by nickoneill** [comment]
Nope, no work started here. AFAIK Android and iOS don't manually change these numbers so we would have to release in tandem with the web frontend (it adds a +1), everything else would work automatically.

**Comment by greggraf** [comment]
I wanted to get comments on the approach I am proposing:

https://github.com/5calls/callserve/compare/master...greggraf:country-code-phone-numbers

1. I added a `test` task to the `Makefile` so I can run the `go test` command
2. I created a `civics_test.go` file with three tests, one for the phone of the official, one for the phone numbers for the field offices, and one for the phone number for the attorney general
3. I updated `formatphone()` to add the +1 prefix and ran the field offices and attorney general phones through them.


One side effect is that I get an error with `make run` because it can't `run` the `civics_test.go` file. 

I am wondering if that task can be changed from 
```
run:
	GOPATH=$(GOPATH) go run *.go $(FIVECALLS_ARGS)
```

to 
```
run:
	GOPATH=$(GOPATH) go run main.go $(FIVECALLS_ARGS)
```

I have not yet updated the front end code.

**Comment by oderby** [comment]
Approach looks more or less fine to me. I'd say go ahead and get that backend work up for code review.

**Comment by oderby** [comment]
Okay - I think all coding is done. #305 is now merged - @nickoneill I'm holding off on merging https://github.com/5calls/callserve/pull/20 because that requires #305 to be deployed first. We also might want to quickly check with android and ios teams to double check that sending +1 prefix won't mess up anything before we merge it. In any event, ball is now in your court ;)
